pandas/tests/arithmetic/test_numeric.py

```diff
@@ -58,6 +58,17 @@ def adjust_negative_zero(zero, expected):
     return expected
 
 
+def compare_op(series, other, op):
```

Comment:
something like this you *might* want o make a first class assert_* function (but ok for now)